tst/com/amazon/ion/benchmark/testNestedStruct.isl

```diff
@@ -3,12 +3,18 @@ schema_header::{
 type::{
     name: Customer,
     type: struct,
-    annotations:[corporate, gold_class, club_member],
```

Comment:
The reason we deleted `annotations` in this test file is in this PR we haven't supported processing constraint `annotations` yet. This task was added to the TODO list and will be supported in the next PR.